assets/src/detour.ts

```diff
@@ -1,8 +1,19 @@
-import { ShapePoint } from "./schedule"
+import { LatLngLiteral } from "leaflet"
+import { Shape, ShapePoint } from "./schedule"
 
 export interface DetourShape {
   coordinates: ShapePoint[]
   directions: {
     instruction: string
   }[]
 }
+
+export interface OriginalRoute {
+  routeName: string
+  routeDescription: string
+  routeOrigin: string
+  routeDirection: string
+  shape: Shape
+  center: LatLngLiteral
```

Comment:
This feels like a case of making wrong code look wrong - in this case the fact that there are leaflet coordinates and schedule `Shape` objects mixed together here.

This was an issue for `StartDetourProps` (where this came from) as well, but it feels more glaring here, since this is lifted up a layer and feels a lot further away from the map itself.

I don't actually want to fix this now, because I think we should decide as a team what the right approach is for dealing with latitude/longitude pairs, and I don't want that discussion to block merging this PR.